source/getting-started/index.markdown

```diff
@@ -9,13 +9,33 @@ sharing: true
 footer: true
 ---
 
+{% comment %}
+
+Note for contributors:
```

Comment:
I have embedded a note to contributors of the getting started section to add some guidelines on how to write a good getting started.